CMakeLists.txt

```diff
@@ -17,6 +17,12 @@ add_custom_target(add-copyright
   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
   COMMENT "Add copyright")
 
+add_custom_target(build-version
```

Comment:
Yes, but that's after I implement `--version`.  This here is enough to stamp binary builds manually when uploading them to a server, for instance.